game/die.cpp

```diff
@@ -0,0 +1,5 @@
+#include "die.h"
+
+int Die::roll () {
```

Comment:
Just put this implementation in the header and then you don't need a `.cpp` file.

Reply:
done.